mmv1/products/beyondcorp/Application.yaml

```diff
@@ -109,6 +115,33 @@ properties:
           description: Optional. Ports of the application.
           item_type:
             type: Integer
+  - name: upstreams
+    type: Array
+    description: Optional. List of which upstream resource(s) to forward traffic to.
+    item_type:
+      type: NestedObject
+      properties:
+        - name: egressPolicy
+          type: NestedObject
+          description: Optional. Routing policy information.
+          properties:
+            - name: regions
+              type: Array
+              description: Required. List of regions where the application sends traffic to.
+              required: true
+              item_type:
+                type: String
+        - name: network
+          type: NestedObject
+          exactly_one_of:
+            - network
+          description: Network to forward traffic to.
+          properties:
+            - name: name
```

Comment:
This should probably be required unless there's a reason to specify a network without a name.